src/main/kotlin/net/atos/zac/flowable/delegate/BerichtenboxDelegate.kt

```diff
@@ -20,8 +20,7 @@ class BerichtenboxDelegate : JavaDelegate {
     }
 
     override fun execute(delegateExecution: DelegateExecution) {
-        LOG.info("Verstuur besluit via Berichtenbox.")
-        LOG.info("VerzendenGelukt = $verzendenGelukt")
+        LOG.info("Send decision via Berichtenbox (Message Box). Send successfully: $verzendenGelukt")
```

Comment:
```suggestion
        LOG.info("Sent decision via Berichtenbox (Message Box). Sent successfully: $verzendenGelukt")
```

The past tense of sending is `sent`. :-)

Why the word `decision` here btw? I don't see anything about a decision in this code?